tests/testthat/test-build_lesson.R

```diff
@@ -165,6 +165,36 @@ test_that("instructor-notes page can be rebuilt", {
 
 })
 
+test_that("empty instructor notes build", {
+  skip_if_not(rmarkdown::pandoc_available("2.11"))
+  # 0. Test that the placeholder text exists in the rendered instructor notes
+  build_instructor_notes(pkg, pages = htmls, quiet = TRUE)
+  expect_true(any(grepl(
+    "This is a placeholder file.",
+    readLines(fs::path(sitepath, "instructor-notes.html"))
+  )))
+  # 1. make a copy of the instructor-notes.md to a local tempfile (use tmp <-
+  # withr::local_tempfile() and fs::file_copy()
+  tmp <- withr::local_tempfile()
+  fs::file_copy(fs::path(res, "instructors/instructor-notes.md"), tmp)
+  # 2. use withr::defer() to do the opposite, copying over the saved file back
+  # when the test finishes)
+  withr::defer({
+    fs::file_copy(tmp, fs::path(res, "instructors/instructor-notes.md"))
```

Comment:
It looks like we forgot to add the `overwrite = TRUE` argument (which you DID include in your first draft, so my bad), which is causing some problems:

```suggestion
    fs::file_copy(tmp, fs::path(res, "instructors/instructor-notes.md"), overwrite = TRUE)
```